articles/azure-cache-for-redis/cache-remove-tls-10-11.md

```diff
@@ -83,8 +84,11 @@ In Java 8, TLS 1.2 is used by default and shouldn't require updates to your clie
 Node Redis and IORedis use TLS 1.2 by default.
 
 ### PHP
-
-Predis on PHP 7 won't work because PHP 7 supports only TLS 1.0. On PHP 7.2.1 or earlier, Predis uses TLS 1.0 or 1.1 by default. You can specify TLS 1.2 when you create the client instance:
+#### Predis
+ < PHP 7 Predis supports only TLS 1.0. Does not work with TLS 1.2, you need to upgrade.
```

Comment:
Let's spell out "Versions less than" because the symbols are HTML characters that will be invisible.